pizzeria-backend/Controllers/OrderController.cs

```diff
@@ -1,30 +1,134 @@
 ﻿using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.SignalR;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Serialization;
+using pizzeria_backend.Hubs;
+using pizzeria_backend.Models.Interfaces;
+using pizzeria_backend.Services;
 
 namespace pizzeria_backend.Controllers
 {
     [Route("api/orders/v1.0")]
     [ApiController]
     public class OrderController : Controller
     {
+        private readonly IOrderService _orderService;
+        private readonly IHubContext<OrderHub> _hubContext;
 
+        public OrderController(IOrderService orderService, IHubContext<OrderHub> hubContext)
+        {
+            _orderService = orderService;
+            _hubContext = hubContext;
+        }
 
         [HttpPost()]
```

Comment:
Since this is not really REST (you have a validate endpoint) you can name this endpoint /create if you think it's better.

Reply:
Done